client/src/components/RecipeCard.js

```diff
@@ -2,55 +2,19 @@ import React, { useState } from "react";
 
 import styled from "styled-components";
 
-export default function RecipeCard({ addRecipeToFavorites }) {
+export default function RecipeCard({ addRecipeToFavorites, recipe }) {
   const [showText, setShowText] = useState(false);
   const onClick = () => setShowText(!showText);
   const Text = () => (
```

Comment:
Isn't this the additional part that is shown once the user clicks on the "Supplies and Steps" button? Look for a more telling variable name like "Details" or even call it "SuppliesAndSteps"